rdrf/rdrf/forms/navigation/quick_links.py

```diff
@@ -85,6 +85,7 @@ class LinkDefs:
     )
     WhitelistedFileExtension = make_link("admin:rdrf_whitelistedfileextension_changelist",
                                          _("Allowed extensions for file uploads"))
+    TotpDevices = make_link("admin:otp_totp_totpdevice_changelist", _("Totp Devices"))
```

Comment:
Is it worth renaming this something a bit more mainstream i.e. Two-factor Auth to align with existing terminology for the TOTP functionality in TRRF

Reply:
We can, but would need to rename the page itself as well. I think it's fine to just include an explanation in the documentation for curators.